examples/image_classification.ipynb

```diff
@@ -910,10 +910,10 @@
     }
```

Comment:
<p>Update to the paragraph:</p><p> </p><p>You might wonder why we pass along the&nbsp;<code>image_processor</code>&nbsp;as a tokenizer when we already preprocessed our data. This is only to make sure the image processor configuration file (stored as JSON) will also be uploaded to the repo on the hub.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/359/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>